models/bloggerLogin.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * This class contains the grantAccess() function which we use to 
+ * determine if the user has entered valid login credentials
+ *
+ * @author jess
+ */
+class bloggerLogin {
```

Comment:
Should we have all our classes starting with capital letters? Apparently that's the stylistic norm?

Reply:
Update this